frontend/src/utils/constants.js

```diff
@@ -169,3 +172,10 @@ export const HeroDataset = {
     priest: priestImg,
     rogue: rogueImg,
 };
+
+export const percentagesToGrade = (percentages) => {
+    if (percentages < 0.5) return 2.0;
+    else {
+        return 3.0 + Math.floor((percentages - 0.5) / 0.1) * 0.5;
```

Comment:
Wyciągnij te cyfry do zmiennych, które opisują znaczenie tych cyfr. Łatwiej będzie sie taki kod czytało. Takie zapisanie jak zrobiłeś nazywa się chyba 'magic numbers' 😉 
(https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/)